Composer/packages/adaptive-form/src/components/AddButton.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import styled from '@emotion/styled';
+import { ActionButton, IButtonStyles } from 'office-ui-fabric-react/lib/components/Button';
+import { FluentTheme } from '@uifabric/fluent-theme';
+import formatMessage from 'format-message';
+
+export const ButtonContainer = styled.div({
+  borderTop: `1px solid ${FluentTheme.palette.neutralLight}`,
+  width: '100%',
+});
+
+export const actionButtonStyles: IButtonStyles = {
+  root: {
+    fontSize: FluentTheme.fonts.small.fontSize,
+    fontWeight: FluentTheme.fonts.small.fontWeight,
+    color: FluentTheme.palette.themePrimary,
+    paddingLeft: 8,
+    height: 20,
+  },
+};
+
+type Props = {
+  onClick: () => void;
+};
+
+export const AddButton = ({ children, onClick }: React.PropsWithChildren<Props>) => {
+  return (
+    <ButtonContainer>
+      <ActionButton styles={actionButtonStyles} onClick={onClick}>
+        {children ?? formatMessage('Add new')}
```

Comment:
This feels a little weird taking arbitrary children; shouldn't this be a string prop (with the same default)?

Reply:
I wanted the component to behave exactly like the `<ActionButton />` which I believe takes arbitrary children.